package.json

```diff
@@ -17,21 +17,21 @@
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "debug": "~2.6.9",
+    "dotenv": "^8.0.0",
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.5.11",
+    "morgan": "^1.9.1",
     "passport": "^0.4.0",
     "passport-jwt": "^4.0.0",
-    "pug": "2.0.0-beta11",
-    "morgan": "^1.9.1"
+    "pug": "2.0.0-beta11"
```

Comment:
why do you need pug?

Reply:
Sorry. I don`t need it. I just took it from example.